qa/vagrant/build.gradle

```diff
@@ -127,21 +131,22 @@ task checkVagrantVersion(type: Exec) {
 }
 
 task vagrantSmokeTest {
-  group 'Vagrant'
-  description 'Smoke test some representative distros from the Vagrantfile'
+  group 'Verification'
+  description 'Smoke test the specified vagrant boxes'
 }
 
-task vagrantSmokeTestAllDistros {
-  group 'Vagrant'
-  description 'Smoke test all distros from the Vagrantfile'
+task packagingTest {
+  group 'Verification'
+  description "Tests yum/apt packages using vagrant and bats.\n" +
+              "    Specify the vagrant boxes to test using the gradle property 'vagrant.boxes'.\n" +
+              "    'sample' can be used to test a single yum and apt box. 'all' can be used to\n" +
+              "    test all available boxes. The available boxes are: \n    ${availableBoxes}"
 }
 
 // Each box gets it own set of tasks
-boxes.each { box ->
-  String boxTask = taskifyBoxName box
-  task "vagrantUp${boxTask}"(type: VagrantCommandTask) {
-    group 'Vagrant'
-    description "Startup a vagrant VM running ${box}"
+for (String box : boxes) {
```

Comment:
Maybe this should be done against availableBoxes and you should conditionally add the dependsOn to vagrantSmokeTest and packagingTest? That way `gradle clean` will try to halt all the boxes.